src/marketplace/Marketplace.js

```diff
@@ -21,31 +71,27 @@ class Marketplace extends React.Component<{||}> {
 				</div>
 				<div className="row mt-5">
 		  			<div className="col-md-4">
-						<h2>Available Tanks</h2>
-						<h6>Tank One</h6>
-						<h6>Tank Two</h6>
-						<h6>Tank Three</h6>
-						<h6>Tank Four</h6>
-						<h6>Tank Five</h6>
-						<h6>Tank Six</h6>
-						<h6>Tank Seven</h6>
-						<h6>Tank Eight</h6>
-		  			</div>
-		  			<div className="col-md-4">
-						<h2 className="text-center">Selected Tank</h2>
+					  <div className="list-group col-md-6">
+						<a className="list-group-item list-group-item-action disabled">Category</a>
```

Comment:
All of the "a" tags generate an error for a broken link. If this is something that can't be fixed until later, then disregard this comment. Otherwise, please fix.

Here is one of the errors:

> The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with the appropriate styles.

Reply:
This is still broken, please address:
![image](https://user-images.githubusercontent.com/18317476/74443139-b2a12480-4e40-11ea-8ac0-d9d405ca9825.png)